core/templates/dev/head/components/entity-creation-services/skill-creation-backend-api.service.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Service for posting the skills
+ */
+
+angular.module('oppia').factory('SkillCreationBackendService', [
```

Comment:
Also, can you move SkillCreationBackendApiService (the new file that was created) to /dev/head/domain/skill? Since the other 'skill' related backend api services, so we can have all of these in one place.